include/messages.h

```diff
@@ -218,8 +218,8 @@
 #define NUMERIC_STR_714      ":%s 714 %s %s :You are already on that channel."
 #define NUMERIC_STR_715      ":%s 715 %s :KNOCKs are disabled."
 #define NUMERIC_STR_716      "%s :is in %s mode and must manually allow you to message them."
-#define NUMERIC_STR_717      "%s :has been informed that you messaged them."
-#define NUMERIC_STR_718      ":%s 718 %s %s %s@%s :is messaging you, and you have umode %s."
+#define NUMERIC_STR_717      "%s :has been informed that you messaged them, but your message has been discarded."
```

Comment:
716 would be a better numeric to modify, as that is always sent. 717 is only sent if the target is notified, which is ratelimited.